### ISSUE TITLE
Drasil Research Group Meeting, Monday, May 13, 3:00 pm, ITB/225

### ISSUE THREAD

**Issue by smiths** [JacquesCarette/Drasil#3730]
0. Admin detail - poster session at the end of the summer
1. Tasks for summer research assistants
- @BilalM04 
- @NoahCardoso 
- @B-rando1 
2. Design discussion
- status of the [redesign of attributes and chunks](https://github.com/JacquesCarette/Drasil/wiki/Attributes-and-Chunks%3a-Meeting-June-7th,-2023)
  - [Explore Space, Symbol and Unit](https://github.com/JacquesCarette/Drasil/projects/20)
  - `nc` / `mkIdea` conundrum (#3314) 
  - Rename chunks named in the form XChunk (#3244, #3733)
  - @balacij if you know other issues related to this discussion, please add them here :smile:
- `constDecDef` (#2179)
- [how does code generation in Drasil work](https://github.com/JacquesCarette/Drasil/issues/3712#issuecomment-2083442644)
- @balacij if there are other design discussion topics that we should have on our radar, please add them here.  We won't get to them in this meeting, but it would be nice to have some idea of topics we should make room to tackle in future design discussions.
4. Next meeting

Regrets: ?

**Comment by balacij** [comment]
Thank you, @smiths ! You read my mind with these :smile:

**Comment by samm82** [comment]
The discussion of "overriding default symbols" reminded me of #1901, although this is more focused on renaming symbols to avoid conflicts instead of to refine them and track their concepts; these two ideas could probably be tackled in parallel?

**Comment by samm82** [comment]
Also, in terms of scheduling next week's meeting, I'm hoping to head home for the long weekend and a bit after, but I will be back for Thursday evening. The later in the week we meet, the more convenient for me, but I can definitely be flexible if we want to meet earlier (even Tuesday)!

**Comment by smiths** [comment]
We are currently looking at Friday next week (May 24) for our next meeting.  Time TBD.
